packages/webcomponents/src/components/business-forms/payment-provisioning/payment-provisioning.tsx

```diff
@@ -23,7 +24,24 @@ export class PaymentProvisioning {
   @State() errorMessage: string = '';
   @State() currentStep: number = 0;
   @State() totalSteps: number = 5;
-  @Event({eventName: 'click-event'}) clickEvent: EventEmitter<BusinessFormClickEvent>;
+  @Event({ eventName: 'click-event' }) clickEvent: EventEmitter<BusinessFormClickEvent>;
```

Comment:
Not really relevant to this PR, but I'm realizing after looking at your code that have `submitted` documented as an event that gets logged on the Payment Provisioning form, but we aren't emitting it at all. I need to fix that in a separate PR. 🛠️